xclim/indices/run_length.py

```diff
@@ -38,7 +37,7 @@ def use_ufunc(
     Parameters
     ----------
     ufunc_1dim: {'from_context', 'auto', True, False}
-    da : xarray.DataArray
+    da : .DataArray
```

Comment:
?

Reply:
Whoops! I was replacing references to `xarray` with `xr` and something went wrong here :)

Reply:
Fixed now.